### PR TITLE
Feature/js demo project

### PR DESCRIPTION
**Pull request by jsanderson44**

### CODE REVIEW

**Review by Neil3079 — APPROVED** [review]

**Review by lucianomarisi — APPROVED** [review]

File: ConfigenDemo/AppEnvironment/AppEnvironment.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// auto-generated by configen
+// to add or remove properties, edit the mapping file: '/Users/johnsanderson/Documents/ConfigGenerator/ConfigenDemo/AppEnvironment.map'.
+// README: https://github.com/theappbusiness/ConfigGenerator/blob/master/README.md
+
+import Foundation
+
+class AppEnvironment {
+
+  let entryPointURL: URL = URL(string: "http://example.com/production")!
```
Comment (samdods) [inline ConfigenDemo/AppEnvironment/AppEnvironment.swift]: looks like there's a bug in the latest code change. these should all be `static`

**Review by samdods — COMMENTED** [review]
nothing wrong with this PR except for the fact that it's highlighted a bug that wasn't picked up in a previous PR. the `let` properties on the config class all need to be `static`

File: ConfigenDemo/AppEnvironment/AppEnvironment.swift (hunk 1 above)
Comment (samdods) [reply ConfigenDemo/AppEnvironment/AppEnvironment.swift]: assuming you can update this once the other PR is merged, right?

**Review by samdods — APPROVED** [review]
this is approved, once the AppEnvironment.swift file is updated with static lets

**Pull request merged**